### PR TITLE
Add docs for Session only cookies

### PR DESCRIPTION
**Pull request by abhi12299**
Pending task for [this PR](https://github.com/gofiber/fiber/pull/1752)

### CODE REVIEW

File: original/context.md [hunk 1]
```diff
@@ -232,14 +232,16 @@ c.Cookie(*Cookie)
...
+	Name        string
+	Value       string
+	Path        string
+	Domain      string
+	MaxAge      int
+	Expires     time.Time
+	Secure      bool
+	HTTPOnly    bool
+	SameSite    string // lax, strict, none
+	SessionOnly bool
```
Comment (ReneWerner87) [inline original/context.md:244]: please don´t change this file, its for the old version

**Review by ReneWerner87 — CHANGES_REQUESTED** [review]
please refresh the csrf documentation

**Comment by abhi12299** [comment]
@ReneWerner87 please review the changes

**Comment by ReneWerner87** [comment]
@abhi12299 looks good so far, can you correct the indentations

**Comment by abhi12299** [comment]
@ReneWerner87 done.

**Review by ReneWerner87 — APPROVED** [review]

**Pull request merged**